src/core/render/compiler.js

```diff
@@ -241,24 +241,32 @@ export class Compiler {
           href = 'README'
         }
         href = router.toURL(href, null, router.getCurrentPath())
-      } else {
-        attrs += href.indexOf('mailto:') === 0 ? '' : ` target="${linkTarget}"`
+      } else if (href.indexOf('mailto:') < 0) {
+        attrs.target = `"${linkTarget}"`
       }
 
       if (config.target) {
-        attrs += ' target=' + config.target
+        attrs.target = `"${config.target}"`
```

Comment:
the important part here is to override the target instead of appending it and having one arbitrarily chosen.